.circleci/config.yml

```diff
@@ -6,27 +6,35 @@ orbs:
 workflows:
   catalog_build:
     jobs:
-      - orb-tools/lint
+      - orb-tools/lint:
+          filters:
+            tags:
+              only: /.*/
 
       - orb-tools/pack:
           source-dir: src
           destination-orb-path: packed/orb.yml
           workspace-path: packed/orb.yml
           artifact-path: packed/orb.yml
           requires: [orb-tools/lint]
+          filters:
+            tags:
+              only: /.*/
 
       - orb-tools/publish-dev:
           orb-name: giantswarm/architect
           orb-path: workspace/packed/orb.yml
           publish-token-variable: CIRCLECI_DEV_API_TOKEN
           requires: [orb-tools/pack]
 
-      - orb-tools/dev-promote-prod:
-          orb-name: giantswarm/architect
+      - orb-tools/publish:
+          attach-workspace: true
+          orb-path: workspace/packed/orb.yml
+          orb-ref: giantswarm/architect@${CIRCLE_TAG##v}
           publish-token-variable: CIRCLECI_DEV_API_TOKEN
-          release: patch
```

Comment:
I have a feeling that this will always bump the patch version, no?

Reply:
We use `orb-tools/publish` and not `orb-tools/dev-promote-prod` here. The difference is that instead of specifying with part to bump, you just pass the version. So I pass the git tag in there.

Reply:
Ah. With you. Awesome!